### DISCUSSION TITLE
Creating Windows Autopilot Virtual Machines on macOS - odds+endpoints

### DISCUSSION THREAD

**Discussion by giscus** [ennnbeee/oddsandendpoints-web#8] [Comments]
# Creating Windows Autopilot Virtual Machines on macOS

Using UTM on macOS to build, configure and deploy a Windows 11 Autopilot Virtual Machine using Microsoft Intune.

https://memv.ennbee.uk/posts/macos-windows-autopilot-utm/

**Comment by abstertee** [comment]
How are you handling any Bitlocker policies?  Do you ever get any errors?

**Reply by ennnbeee** [reply]
You know what, I don't think I applied a BitLocker policy to the device, though I imagine without there being a physical TPM or even a virtual one available, that any BitLocker policy that requires one, will likely fail. 

I guess for testing you could not have that setting turned on.

**Reply by abstertee** [reply]
I do see that they can now emulate the TPM chip, however I am getting an error when applying the BitLocker policy (0x800705b4).  This could be the policy failing.  I'll try testing without the BitLocker policy and see what happens.

**Reply by ennnbeee** [reply]
@abstertee I've had chance to test, and the device will happily encrypt with BitLocker. 

Just check if the UTM Guest tools ISO or any other ISO is attached to the VM, as this will stop automatic/silent BitLocker encryption.
